VERSION

```diff
@@ -0,0 +1 @@
+v3.1.0-48-g109a6c8
```

Comment:
Do we still need this file if the github action creates it?

Reply:
nope, will remove